doc/developer.md

```diff
@@ -205,3 +205,14 @@ For the Rust core library we use `rMAJOR.MINOR.PATH`
 (note it starts with `r`, and not `v`).
 The Rust version is not automated,
 and must be bumped in `src/core/Cargo.toml`.
+
```

Comment:
Note that this TOC ends up at the bottom of the file. Is this intentional (and if so, why 😂)

Reply:
I collected orphan links into a toc, but opted to put it in the bottom to avoid disrupting the current docs. But it might be more useful to put on the top...